cmd/tetra/getevents/getevents.go

```diff
@@ -36,18 +36,19 @@ type Opts struct {
 	Timestamps    bool
```

Comment:
could you please some example output into changelog? generally all the commits should have some description so one does not need to parse the diff to get an idea ;-) thanks

Reply:
Yes, sure!